pkg/cmd/server/start/start_master.go

```diff
@@ -529,6 +530,10 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		return nil
 	}
 
+	if sdnplugin.IsOpenShiftNetworkPlugin(oc.Options.NetworkConfig.NetworkPluginName) && oc.Options.ControllerLeaseTTL <= 0 {
```

Comment:
If you're in this method, you're a controller and running with a block.  You can't run networking outside of this.